LoopBackTests/server/index.js

```diff
@@ -6,15 +6,11 @@
 var loopback = require('loopback');
 var path = require('path');
 var app = loopback();
-app.set('legacyExplorer', false);
-app.use(loopback.logger(app.get('env') === 'development' ? 'dev' : 'default'));
-app.dataSource('Memory', {
-  connector: loopback.Memory,
-  defaultForType: 'db'
-});
 
-var Widget = app.model('widget', {
-  properties: {
+app.dataSource('Memory', { connector: 'memory' });
```

Comment:
nitpick (feel free to ignore): we usually use `db` as the name of the main datasource.